modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -567,6 +567,43 @@ def fix_model_input_data(model_data, model_class):
             )
     return fixed_model_data
 
+def get_discriminator_class(model_class, from_server, model_data, cls_visited):
+    """Returns the child class specified by the discriminator.
+
+    Args:
+        model_class (OpenApiModel): the model class
+        from_server (bool): True if the model_data is from the server
+            False if the data is from the client
+        model_data (list/dict): data to instantiate the model.
+        cls_visited (set): list of model classes that have been visited.
+            Used to determine the discriminator class without
+            visiting circular references indefinitely.
+
+    Returns:
+        A tuple containing the child class specified by the discriminator,
+        the name of the discriminator property, and the discriminator value
+        in the input data.
```

Comment:
Can you add type descriptions here?
```
A tuple containing (used_model_class, discr_propertyname_js, discr_value)
used_model_class (class): the chosen class that will be used to deserialize the data, for example dog.Dog
discr_propertyname_js (string): the discriminator.propertyName from the opeanpi spec
discr_value (string): the class name in model_data which is the key that we use to return used_model_class from our discriminator, for example 'Dog'
```

Reply:
Sure, I am adding. We should use the term "OpenAPI document" not "OpenAPI specification" when referring to a YAML/JSON document describing an API. The "spec" is the standard.